src/System.Windows.Forms.Primitives/tests/UnitTests/Interop/PARAMTests.cs

```diff
@@ -40,85 +40,85 @@ public unsafe void FromLowHighUnsigned_x64_Result()
         [ConditionalFact(typeof(ArchitectureDetection), nameof(ArchitectureDetection.Is32bit))]
         public unsafe void LOWORD_x32_Result()
         {
-            Assert.Equal(0x0304, PARAM.LOWORD((IntPtr)unchecked((int)0x01020304)));
-            Assert.Equal(0xF3F4, PARAM.LOWORD((IntPtr)unchecked((int)0xF1F2F3F4)));
+            Assert.Equal(0x0304, PARAM.LOWORD((nint)(int)0x01020304));
+            Assert.Equal(0xF3F4, PARAM.LOWORD((nint)unchecked((int)0xF1F2F3F4)));
         }
 
         [ConditionalFact(typeof(ArchitectureDetection), nameof(ArchitectureDetection.Is64bit))]
         public unsafe void LOWORD_x64_Result()
         {
-            Assert.Equal(0x0304, PARAM.LOWORD((IntPtr)unchecked((long)0x0506070801020304)));
-            Assert.Equal(0xF3F4, PARAM.LOWORD((IntPtr)unchecked((long)0xF5F6F7F8F1F2F3F4)));
+            Assert.Equal(0x0304, PARAM.LOWORD(unchecked((nint)(long)0x0506070801020304)));
+            Assert.Equal(0xF3F4, PARAM.LOWORD(unchecked((nint)unchecked((long)0xF5F6F7F8F1F2F3F4))));
```

Comment:
This looks like an overkill.... Should these be?
```suggestion
            Assert.Equal(0x0304, PARAM.LOWORD(unchecked((nint)0x0506070801020304)));
            Assert.Equal(0xF3F4, PARAM.LOWORD(unchecked((nint)0xF5F6F7F8F1F2F3F4)));
```